client/my-sites/checklist/controller/index.jsx

```diff
@@ -5,29 +5,15 @@
  */
 
 import React from 'react';
-import ReactDom from 'react-dom';
 
 /**
  * Internal Dependencies
  */
 
-import { renderWithReduxStore } from 'lib/react-helpers';
-import { setSection } from 'state/ui/actions';
 import ChecklistShow from '../checklist-show';
-import ChecklistThankYou from '../checklist-thank-you';
```

Comment:
If we no longer use the component, why don't we remove the file completely?

Reply:
Yes - I plan to remove it - just thought it'd be easier to leave it for the moment for comparison with the style and content of the original component